rpm/podman.spec

```diff
@@ -37,7 +37,7 @@
 # include it. Official rawhide should be able to fetch the last active build of
 # gvproxy, the min version requirement has been removed to allow it.
 # Ref: https://bugzilla.redhat.com/show_bug.cgi?id=2224434
-%if !%{defined copr_username} && 0%{?fedora} <= 38
+%if !%{defined copr_username} && 0%{?fedora} <= 37
```

Comment:
is this version down grade correct?

Reply:
yes, that's intentional. gvproxy is an independent package for Fedora 38 and higher.